misc/python/materialize/cli/mzconduct.py

```diff
@@ -708,6 +708,48 @@ def run(self, comp: Composition, workflow: Workflow) -> None:
             say(f"INFO: error purging topics: {e}")
 
 
+@Steps.register("pause-container")
+class PauseContainerStep(WorkflowStep):
+    """Pauses and unpauses the designated Docker container.
+
+    Params:
+        container: The Docker container to pause
+        loop: True to continually pause and unpause, False to pause and unpause once
+        wait: The number of seconds to wait before sleeping (default: 30)
+        sleep: The number of seconds to pause for (default: 30)
```

Comment:
I think these might be slightly more clear as:

* `loop_time`: amount of time to spend pausing and un-pausing
* `paused_time`: per-iteration time to spend while the named service is paused
* `running_time`: per-iteration time to spend with the named service running